modules/tile-converter/src/converter-cli.ts

```diff
@@ -16,6 +16,8 @@ type TileConversionOptions = {
   output: string;
   /** Keep created 3DNodeIndexDocument files on disk instead of memory. This option reduce memory usage but decelerates conversion speed */
   instantNodeWriting: boolean;
+  /** Try to merge similar materials to be able to merge meshes into one node (I3S to 3DTiles conversion only) */
+  mergeMaterials: boolean;
```

Comment:
Don't you want these to be optional?

```suggestion
  mergeMaterials?: boolean;
```

Reply:
No